src/dashboard/src/templates/backlog/delete_request.html

```diff
@@ -0,0 +1,21 @@
+{% extends "layout_fluid.html" %}
```

Comment:
This is almost identical to `archival_storage/delete_request.html`.  Can `archival_storage/delete_request.html` be moved to the `templates` dir and reused? Seems like a useful common template.